### DISCUSSION TITLE
lockFileMaintenance shows wrong packageName in PR title

### DISCUSSION THREAD

**Discussion by nitrocode** [renovatebot/renovate#19969] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Here is the PR created by renovatebot

- https://github.com/runatlantis/atlantis/pull/2987

The original title was

```
chore(deps): lock file maintenance in server/events/testdata/test-repos/workspace-configured/main.tf
```

But it does not update `server/events/testdata/test-repos/workspace-configured/main.tf` and instead updates `yarn.lock` but for some reason `yarn.lock` isn't populated by `packageFile`

https://github.com/runatlantis/atlantis/blob/02b29e6b087bd21a439fed4d6e83b0ea7d3c4434/.github/renovate.json5#L6

Here is the dependency dashboard showing another pr with the wrong packageFile

https://github.com/runatlantis/atlantis/issues/2818

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by rarkins** [comment]
Converting to a discussion, because:
- this is caused by config
- can be resolved in config
- this config item is marked as "use at your own risk"

I think you'll find the same probably any time you have multiple or attempted updates in a PR from different package files.

You might not have that scenario for regular updates, so in that case you could avoid this in lock file maintenance by configuring it in a package rule which applies to only patch, minor, major updates.

**Reply by nitrocode** [reply]
So this isn't a bug?

**Reply by rarkins** [reply]
It's not

**Reply by nitrocode** [reply]
So the solution is to not use lockFileMaintenance with packageFile in the pr title due to it being marked as "use at your own risk"?

I'm not sure i fully understand how to resolve this other than to just disable lockFileMaintenance?

**Reply by rarkins** [reply, marked as answer]
The solution is to stop using commitMessageSuffix for any scenarios where you have more than one file updated in the same PR, and also stop using it altogether for lock file maintenance.

**Reply by nitrocode** [reply]
Hmm but there is only one file updated in my associated PR https://github.com/runatlantis/atlantis/pull/2987 and the file name that is was in the PR title (prior to my manual edit) was not the file that was modified.

Thank you for the quick responses!
